### PR TITLE
Reverse individual words of a string using stacks in C (issue #2585)

### PR DESCRIPTION
**Pull request by taruni-always**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes
### Description

Implementation of reversing individual words of a string using stacks in C language.
(Issue #2585)
I wanted to contribute under GSSoC '21.
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
(issue #2585)

In reference with : #2900

### CODE REVIEW

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top

**Review by ankitaggarwal23 — APPROVED** [review]

File: C/ds/reverseWordsOfString.c [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    /* calling the reverseWords function */
+    return 0;
+}
+
+/*
+Sample Input1: Get me food
+Sample Output1: teG em doof
+
+Sample Input2: happy birthday
+Sample Output2: yppah yadhtrib
```
Comment (ankitaggarwal23) [inline C/ds/reverseWordsOfString.c:83]: Code produces output like that change it and remove the extra test case one is enough

Comment (ankitaggarwal23) [inline C/ds/reverseWordsOfString.c:83]: Code produces output like that change it and remove the extra test case one is enough

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C/ds/reverseWordsOfString.c [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+*/
```
Comment (ankitaggarwal23) [inline C/ds/reverseWordsOfString.c]: Code produces output like that change it and remove the extra test case one is enough.

Capture

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by taruni-always** [comment]
Apart from removing the extra test case, what change am I supposed to make @ankitaggarwal23 ?

**Review by ankitaggarwal23 — COMMENTED** [review]
Kindly change the output I added the snapshot.

**Comment by taruni-always** [comment]
image
Is this how you want the output to be?  (I mean without any prompts)?

**Comment by ankitaggarwal23** [comment]
> image
> Is this how you want the output to be? (I mean without any prompts)?

I added the screenshot. Your code producing like that

**Comment by ankitaggarwal23** [comment]
@taruni-always You have to write the output the way your code is producing See, How your code is producing Output with proper display message and how you wrote previously Kindly rewrite it

**Comment by taruni-always** [comment]
@ankitaggarwal23 got it! Thanks will do that. :-)

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**